src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = days * 40;
```

Comment:
Consider using a constant for the daily rental cost (40) for better readability and easier maintenance. 📖